hack/cloudbuild.sh

```diff
@@ -10,7 +10,7 @@ echo "PLATFORM: ${PLATFORM}"
 # debug the rest of the script in case of image/CI build issues
 set -o xtrace
 
-REPO="gcr.io/k8s-staging-sig-storage"
+REPO="registry.k8s.io/k8s-staging-sig-storage"
```

Comment:
I don't think this is going to work.  registry.k8s.io doesn't have a web interface to browse images, but I can browse it with this: https://explore.ggcr.dev/?repo=registry.k8s.io

There is no `k8s-staging-sig-storage` in that list. Only the non-staging `sig-storage` repo. What are we supposed to do for staging images?